---
layout: page
title: Publications
permalink: /publications/
author_profile: true
mathjax: true
---

See also my [Google Scholar profile](https://scholar.google.com/citations?user=ths6zX0AAAAJ&hl=en).

#### Patents

1. Oligonucleotides for modulating RTEL1 expression. (2020)
2. Pyrrolo[2,3-b]pyrazine compounds as cccDNA inhibitors for the treatment of Hepatitis B Virus (HBV) infection. (2018)

#### Book chapters

1. Edited by Adetayo Kasim, S. H., Ziv Shkedy, Sebastian Kaiser & Talloen, W.
   Applied Biclustering Methods for Big and High-Dimensional Data Using R
   (Chapman & Hall/CRC Biostatistics Series). Chapman and Hall/CRC (2016).
   ([Publisher's
   Link](https://www.routledge.com/Applied-Biclustering-Methods-for-Big-and-High-Dimensional-Data-Using-R/Kasim-Shkedy-Kaiser-Hochreiter-Talloen/p/book/9781482208238),
   [Amazon](https://www.amazon.com/Applied-Biclustering-Methods-High-Dimensional-Biostatistics/dp/1482208237)).

#### Preprints

1. Fang, T., Davydov, I., Marbach, D. & Zhang, J. D. [Gene-set enrichment with
   regularized regression](https://www.biorxiv.org/content/10.1101/659920v2) (2019).
2. Zhang, Y. *et al.* [A Novel, Anatomy-Similar in Vitro Model of 3D Airway
   Epithelial for Anti-Coronavirus Drug
   Discovery](https://doi.org/10.1101/2021.03.03.433824) (2021)
3. Geser, P. and Zhang, J. D. [Gene symbol recognition with GeneOCR](https://www.biorxiv.org/content/10.1101/2022.07.01.498459v3) (2022)


#### Peer-reviewed publications

##### 2022

46. Chen, D. *et al.* [Discovery of Novel cccDNA Reducers toward the Cure of Hepatitis B Virus Infection](https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c02215), **Journal of Medicinal Chemistry** (2022)
45. Gatti, L. *et al.* [Cross-reactive immunity potentially drives global oscillation and opposed alternation patterns of seasonal influenza A viruses](https://www.nature.com/articles/s41598-022-08233-w), **Scientific Reports** (2022)
44. Jaklin, M. *et al.* [Optimization of the *TeraTox* assay for preclinical teratogenicity assessment](https://academic.oup.com/toxsci/advance-article/doi/10.1093/toxsci/kfac046/6575922), **Toxicological Sciences** (2022)

##### 2021

43. Mädler, S. *et al.* [Besca, a single-cell transcriptomics analysis toolkit to accelerate translational research](https://academic.oup.com/nargab/article/3/4/lqab102/6423167), **NAR Genomics and Bioinformatics** (2021)
42. Sturm, G. *et al.* [Tissue heterogeneity is prevalent in gene expression studies](https://academic.oup.com/nargab/article/3/3/lqab077/6363752), **NAR Genomics and Bioinformatics** (2021)
41. Zhang, J. D. [Ten Simple Rules for Doing A Postdoc In Pharma](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008989), **PLOS Computational Biology** (2021)
40. Reich, *et al.* [Alzheimer’s Risk Gene TREM2 Determines Functional Properties of New Type of Human iPSC-Derived Microglia](https://www.frontiersin.org/articles/10.3389/fimmu.2020.617860/full), **Frontiers in Immunology** (2021)

##### 2020

39. Jaklin, *et al.* [Focus on germ-layer markers: A human stem cell-based model for in vitro teratogenicity testing](https://doi.org/10.1016/j.reprotox.2020.10.011).
    **Reproductive Toxicology** (2020)
38. Roudnicky, F. *et al.* [Inducers of the endothelial cell barrier identified
    through chemogenomic screening in genome-edited hPSC-endothelial
    cells](https://www.pnas.org/content/early/2020/08/04/1911532117).
    **Proceedings of the National Academy of Sciences** (2020)
37. Gutbier, S. *et al.* [Large-Scale Production of Human iPSC-Derived Macrophages for Drug Screening](https://www.mdpi.com/1422-0067/21/13/4808). **International Journal of Molecular Sciences** (2020)
36. Zaidan, M. *et al.* [Signaling pathways predisposing to chronic kidney
    disease progression](https://insight.jci.org/articles/view/126183). **JCI Insight** (2020)
35. Badillo, S. *et al.* [An Introduction to Machine Learning](https://ascpt.onlinelibrary.wiley.com/doi/full/10.1002/cpt.1796). **Clinical Pharmacology & Therapeutics** (2020).

##### 2019
34. Zhang, J. D., Sach-Peltason, L., Kramer, C., Wang, K. & Ebeling, M. Multiscale modelling of drug mechanism and safety. **Drug Discovery Today** (2019).
33. Sturm, G. *et al.* Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology. **Bioinformatics** 35, i436--i445 (2019).
32. Roudnicky, F. *et al.* Modeling the Effects of Severe Metabolic Disease by Genome Editing of hPSC-Derived Endothelial Cells Reveals an Inflammatory Phenotype. **International Journal of Molecular Sciences** 20, 6201 (2019).
31. Choobdar, S. *et al.* Assessment of network module identification across complex diseases. **Nature Methods** 16, 843--852 (2019).

##### 2018
30. Mueller, H. *et al.* A novel orally available small molecule that inhibits hepatitis B virus expression. **Journal of Hepatology** 68, 412--420 (2018).

##### 2017
29. Zhang, J. D. *et al.* Detect tissue heterogeneity in gene expression data with BioQC. **BMC Genomics** 18, 277 (2017).
28. Moisan, A. *et al.* Inhibition of EGF Uptake by Nephrotoxic Antisense Drugs In Vitro and Implications for Preclinical Safety Profiling. **Molecular Therapy-Nucleic Acids** 6, 89--105 (2017).
27. Drawnel, F. M. *et al.* Molecular Phenotyping Combines Molecular Information, Biological Relevance, and Patient Data to Improve Productivity of Early Drug Discovery. **Cell Chemical Biology** 24, 624--634 (2017).
26. Boess, F. *et al.* Use of early phenotypic in vivo markers to assess human relevance of an unusual rodent non-genotoxic carcinogen in vitro. **Toxicology** 379, 48--61 (2017).

##### 2016
25. Raza, U. *et al.* The miR-644a/CTBP1/p53 axis suppresses drug resistance by simultaneous inhibition of cell survival and epithelial-mesenchymal transition in breast cancer. **Oncotarget** (2016).
24. Grabole, N. *et al.* Genomic analysis of the molecular neuropathology of tuberous sclerosis using a human stem cell model. **Genome Medicine** 8, 94 (2016).

##### 2015
23. Zhang, J. D., Küng, E., Boess, F., Certa, U. & Ebeling, M. Pathway reporter genes define molecular phenotypes of human cells. **BMC Genomics** 16, 342 (2015).
22. Xu, J. *et al.* 14-3-3$\zeta$ turns TGF-$\beta$'s function from tumor suppressor to metastasis promoter in breast cancer by contextual changes of Smad partners from p53 to Gli2. **Cancer Cell** 27, 177--192 (2015).
21. van der Vries, E. *et al.* Outcomes and susceptibility to neuraminidase inhibitors in individuals infected with different influenza B lineages: the influenza resistance information study. **The Journal of Infectious Diseases** 213, 183--190 (2015).
20. Moisan, A. *et al.* White-to-brown metabolic conversion of human adipocytes by JAK inhibition. **Nature Cell Biology** 17, 57--67 (2015).
19. Lenz, N. *et al.* Antiviral Innate Immune Activation in HIV-Infected Adults Negatively Affects H1/IC31-Induced Vaccine-Specific Memory CD4+ T Cells. **Clinical and Vaccine Immunology** 22, 688--696 (2015).
18. Haller, F. *et al.* Combined DNA methylation and gene expression profiling in gastrointestinal stromal tumors reveals hypomethylation of SPP1 as an independent prognostic factor. **International Journal of Cancer** 136, 1013--1023 (2015).

##### 2014
17. Zhang, J. D., Schindler, T., Küng, E., Ebeling, M. & Certa, U. Highly sensitive amplicon-based transcript quantification by semiconductor sequencing. **BMC Genomics** 15, 565 (2014).
16. Zhang, J., Berntenis, N., Roth, A. & Ebeling, M. Data mining reveals a network of early-response genes as a consensus signature of drug-induced in vitro and in vivo toxicity. **The Pharmacogenomics Journal** 14, 208--216 (2014).
15. Yuk, I. H. *et al.* Effects of copper on CHO cells: insights from gene expression analyses. **Biotechnology Progress** 30, 429--442 (2014).
14. Rehman, S. K. *et al.* 14-3-3$\zeta$ Orchestrates Mammary Tumor Onset and Progression via miR-221--Mediated Cell Proliferation. **Cancer Research** 74, 363--373 (2014).
13. Raza, U., Zhang, J. D. & Sahin, Ö. MicroRNAs: master regulators of drug resistance, stemness, and metastasis. **Journal of Molecular Medicine** 92, 321--336 (2014).
12. Aigner, S., Heckel, T., Zhang, J. D., Andreae, L. C. & Jagasia, R. Human pluripotent stem cell models of autism spectrum disorder: emerging frontiers, opportunities, and challenges towards neuronal networks in a dish. **Psychopharmacology** 231, 1089--1104 (2014).

##### 2013
11. Ward, A. *et al.* Re-expression of microRNA-375 reverses both tamoxifen resistance and accompanying EMT-like properties in breast cancer. **Oncogene** 32, 1173--1182 (2013).
10. Horvát, E.-Á., Zhang, J. D., Uhlmann, S., Sahin, Ö. & Zweig, K. A. A network-based method to assess the statistical significance of mild co-regulation effects. **PLOS One** 8, e73413 (2013).
9. Adam, L. *et al.* Plasma microRNA profiles for bladder cancer detection. **Urologic Oncology: Seminars and Original Investigations** vol. 31 1701--1708 (Elsevier, 2013).

##### 2012
8. Ward, A. *et al.* Abstract A14: Re-expression of microRNA-375 reverses both tamoxifen resistance and accompanying EMT-like properties in breast cancer. **Clinical Cancer Research** 18, A14--A14 (2012).
7. Uhlmann, S. *et al.* Global miRNA regulation of a local protein network: Case study with the EGFR-driven cell cycle network in breast cancer. **Molecular Systems Biology** 8, 570 (2012).
6. Keklikoglou, I. *et al.* MicroRNA-520/373 family functions as a tumor suppressor in estrogen receptor negative breast cancer by targeting NF-kappaB and TGF-ß signaling pathways. **Oncogene** 31, 4150 (2012).
5. Jurmeister, S. *et al.* MicroRNA-200c represses migration and invasion of breast cancer cells by targeting actin-regulatory proteins FHOD1 and PPM1F. **Molecular and Cellular Biology** 32, 633--651 (2012).

##### 2011
4. Zhang, J. D. *et al.* Time-resolved human kinome RNAi screen identifies a network regulating mitotic-events as early regulators of cell proliferation. **PLOS One** 6, e22176 (2011).

##### 2010
3. Uhlmann, S. *et al.* miR-200bc/429 cluster targets PLC$\gamma$1 and differentially regulates proliferation and EGF-driven invasion than miR-200a/141 in breast cancer. **Oncogene** 29, 4297--4306 (2010).
2. Haller, F. *et al.* Localization-and mutation-dependent microRNA (miRNA) expression signatures in gastrointestinal stromal tumours (GISTs), with a cluster of co-expressed miRNAs located at 14q32. 31. **The Journal of Pathology** 220, 71-86 (2010).

##### 2009
1. Zhang, J. D. & Wiemann, S. KEGGgraph: a graph approach to KEGG PATHWAY in R and Rioconductor. **Bioinformatics** 25, 1470-1471 (2009).
