---
layout: page
title: Research
permalink: /research/
author_profile: true
mathjax: true
---

<p id="top"></p>
- [Peer-reviewed publications](#publications)
- [Reviews](#reviews)
- [Preprints](#preprints)
- [Patents](#patents)
- [Book chapters](#book)
- [Conference posters and papers](#conference)
- [Essays](#essays)
- [Software](#software)

<p id="publications"></p>
#### Peer-reviewed publications

Most publications are accessible from [my Google Scholar profile](https://scholar.google.com/citations?user=ths6zX0AAAAJ&hl=en).

* Rot, G. *et al.* [*splicekit*: an integrative toolkit for splicing analysis from short-read RNA-seq](https://academic.oup.com/bioinformaticsadvances/article/4/1/vbae121/7735317) (2024)
* Bosch, M. *et al.* [A liver immune rheostat regulates CD8 T cell immunity in chronic HBV infection](https://doi.org/10.1038/s41586-024-07630-7) (2024).
* Courzet, S. *et al*. [G-PLIP: Knowledge graph neural network for structure-free protein-ligand affinity prediction](https://doi.org/10.1016/j.csbj.2024.06.029) (2024).
* Klughammer, B. *et al.* [A randomized, double-blind phase 1b study evaluating the safety, tolerability, pharmacokinetics and pharmacodynamics of the NLRP3 inhibitor selnoflast in patients with moderate to severe active ulcerative colitis](https://onlinelibrary.wiley.com/doi/10.1002/ctm2.1471) (2023).
* Wang, L. *et al.* [Discovery of a first-in-class orally available HBV cccDNA inhibitor](https://www.sciencedirect.com/science/article/pii/S0168827822034663) (2023)
* Chen, D. *et al.* [Discovery of Novel cccDNA Reducers toward the Cure of Hepatitis B Virus Infection](https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c02215) (2022)
* Gatti, L. *et al.* [Cross-reactive immunity potentially drives global oscillation and opposed alternation patterns of seasonal influenza A viruses](https://www.nature.com/articles/s41598-022-08233-w) (2022)
* Jaklin, M. *et al.* [Optimization of the *TeraTox* assay for preclinical teratogenicity assessment](https://academic.oup.com/toxsci/advance-article/doi/10.1093/toxsci/kfac046/6575922) (2022)
* Mädler, S. *et al.* [Besca, a single-cell transcriptomics analysis toolkit to accelerate translational research](https://academic.oup.com/nargab/article/3/4/lqab102/6423167) (2021)
* Sturm, G. *et al.* [Tissue heterogeneity is prevalent in gene expression studies](https://academic.oup.com/nargab/article/3/3/lqab077/6363752) (2021)
* Zhang, J. D. [Ten Simple Rules for Doing A Postdoc In Pharma](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008989) (2021)
* Reich, *et al.* [Alzheimer’s Risk Gene TREM2 Determines Functional Properties of New Type of Human iPSC-Derived Microglia](https://www.frontiersin.org/articles/10.3389/fimmu.2020.617860/full) (2021)
* Jaklin, *et al.* [Focus on germ-layer markers: A human stem cell-based model for in vitro teratogenicity testing](https://doi.org/10.1016/j.reprotox.2020.10.011) (2020)
* Roudnicky, F. *et al.* [Inducers of the endothelial cell barrier identified through chemogenomic screening in genome-edited hPSC-endothelial cells](https://www.pnas.org/content/early/2020/08/04/1911532117) (2020)
* Gutbier, S. *et al.* [Large-Scale Production of Human iPSC-Derived Macrophages for Drug Screening](https://www.mdpi.com/1422-0067/21/13/4808) (2020)
* Zaidan, M. *et al.* [Signaling pathways predisposing to chronic kidney disease progression](https://insight.jci.org/articles/view/126183) (2020)
* Sturm, G. *et al.* [Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology](https://doi.org/10.1093/bioinformatics/btz363) (2019).
* Roudnicky, F. *et al.* [Modeling the Effects of Severe Metabolic Disease by Genome Editing of hPSC-Derived Endothelial Cells Reveals an Inflammatory Phenotype](https://www.mdpi.com/1422-0067/20/24/6201) (2019)
* Choobdar, S. *et al.* [Assessment of network module identification across complex diseases](https://www.nature.com/articles/s41592-019-0509-5) (2019).
* Mueller, H. *et al.* [A novel orally available small molecule that inhibits hepatitis B virus expression](https://www.sciencedirect.com/science/article/pii/S0168827817323930) (2018).
* Zhang, J. D. *et al.* [Detect tissue heterogeneity in gene expression data with BioQC](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3661-2) (2017).
* Moisan, A. *et al.* [Inhibition of EGF Uptake by Nephrotoxic Antisense Drugs In Vitro and Implications for Preclinical Safety Profiling](https://doi.org/10.1016/j.omtn.2016.11.006) (2017).
* Drawnel, F. M. *et al.* [Molecular Phenotyping Combines Molecular Information, Biological Relevance, and Patient Data to Improve Productivity of Early Drug Discovery](https://doi.org/10.1016/j.chembiol.2017.03.016) (2017).
* Boess, F. *et al.* [Use of early phenotypic in vivo markers to assess human relevance of an unusual rodent non-genotoxic carcinogen in vitro](https://doi.org/10.1016/j.tox.2017.01.018) (2017).
* Raza, U. *et al.* [The miR-644a/CTBP1/p53 axis suppresses drug resistance by simultaneous inhibition of cell survival and epithelial-mesenchymal transition in breast cancer](https://www.ncbi.nlm.nih.gov/pmc/articles/pmid/27409664/) (2016).
* Grabole, N. *et al.* [Genomic analysis of the molecular neuropathology of tuberous sclerosis using a human stem cell model](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0347-3) (2016).
* Zhang, J. D. *et al.* [Pathway reporter genes define molecular phenotypes of human cells](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1532-2) (2015).
* Xu, J. *et al.* [14-3-3$\zeta$ turns TGF-$\beta$'s function from tumor suppressor to metastasis promoter in breast cancer by contextual changes of Smad partners from p53 to Gli2](https://linkinghub.elsevier.com/retrieve/pii/S1535-6108(14)00476-0) (2015).
* van der Vries, E. *et al.* [Outcomes and susceptibility to neuraminidase inhibitors in individuals infected with different influenza B lineages: the influenza resistance information study](https://academic.oup.com/jid/article-lookup/doi/10.1093/infdis/jiv375) (2015).
* Moisan, A. *et al.* [White-to-brown metabolic conversion of human adipocytes by JAK inhibition](https://www.ncbi.nlm.nih.gov/pmc/articles/pmid/25487280/) (2015).
* Lenz, N. *et al.* [Antiviral Innate Immune Activation in HIV-Infected Adults Negatively Affects H1/IC31-Induced Vaccine-Specific Memory CD4+ T Cells](https://pubmed.ncbi.nlm.nih.gov/25924764/) (2015).
* Haller, F. *et al.* [Combined DNA methylation and gene expression profiling in gastrointestinal stromal tumors reveals hypomethylation of SPP1 as an independent prognostic factor](https://pubmed.ncbi.nlm.nih.gov/25046773/) (2015).
* Zhang, J. D. *et al.* [Highly sensitive amplicon-based transcript quantification by semiconductor sequencing](https://pubmed.ncbi.nlm.nih.gov/24997760/) (2014).
* Zhang, J. D. *et al.* [Data mining reveals a network of early-response genes as a consensus signature of drug-induced in vitro and in vivo toxicity](https://pubmed.ncbi.nlm.nih.gov/24217556/) (2014).
* Yuk, I. H. *et al.* [Effects of copper on CHO cells: insights from gene expression analyses](https://pubmed.ncbi.nlm.nih.gov/24403277/) (2014).
* Rehman, S. K. *et al.* [14-3-3$\zeta$ Orchestrates Mammary Tumor Onset and Progression via miR-221--Mediated Cell Proliferation](https://pubmed.ncbi.nlm.nih.gov/24197133/) (2014).
* Ward, A. *et al.* [Re-expression of microRNA-375 reverses both tamoxifen resistance and accompanying EMT-like properties in breast cancer](https://doi.org/10.1038/onc.2012.128) (2013).
* Horvát, E.-Á., Zhang, J. D., Uhlmann, S., Sahin, Ö. & Zweig, K. A. [A network-based method to assess the statistical significance of mild co-regulation effects](https://dx.plos.org/10.1371/journal.pone.0073413) (2013).
* Adam, L. *et al.* [Plasma microRNA profiles for bladder cancer detection](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5226073/) (2013).
* Uhlmann, S. *et al.* [Global microRNA level regulation of EGFR-driven cell-cycle protein network in breast cancer](https://doi.org/10.1038/msb.2011.100) (2012).
* Keklikoglou, I. *et al.* [MicroRNA-520/373 family functions as a tumor suppressor in estrogen receptor negative breast cancer by targeting NF-kappaB and TGF-ß signaling pathways](https://www.nature.com/articles/onc2011571) (2012).
* Jurmeister, S. *et al.* [MicroRNA-200c represses migration and invasion of breast cancer cells by targeting actin-regulatory proteins FHOD1 and PPM1F](https://www.tandfonline.com/doi/full/10.1128/MCB.06212-11) (2012).
* Zhang, J. D. *et al.* [Time-resolved human kinome RNAi screen identifies a network regulating mitotic-events as early regulators of cell proliferation](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0022176) (2011).
* Uhlmann, S. *et al.* [miR-200bc/429 cluster targets PLC$\gamma$1 and differentially regulates proliferation and EGF-driven invasion than miR-200a/141 in breast cancer](https://www.nature.com/articles/onc2010201) (2010).
* Haller, F. *et al.* [Localization-and mutation-dependent microRNA (miRNA) expression signatures in gastrointestinal stromal tumours (GISTs), with a cluster of co-expressed miRNAs located at 14q32.31](https://pathsocjournals.onlinelibrary.wiley.com/doi/10.1002/path.2610) (2010).
* Zhang, J. D. & Wiemann, S. [KEGGgraph: a graph approach to KEGG PATHWAY in R and Rioconductor](https://academic.oup.com/bioinformatics/article/25/11/1470/330944) (2009).
{: reversed="reversed"}

<p id="reviews"></p>
<a href="#top" style="font-size: 11px;">Back to top</a>
#### Reviews

* Bassani, *et al.* [Another string to your bow: machine learning prediction of the pharmacokinetic properties of small molecules](https://www.tandfonline.com/doi/full/10.1080/17460441.2024.2348157) (2024).
* Michoel, T. and Zhang, J. D. [Causal inference in drug discovery and development](https://www.sciencedirect.com/science/article/pii/S1359644623002532) (2023)
* Badillo, S. *et al.* [An Introduction to Machine Learning](https://ascpt.onlinelibrary.wiley.com/doi/full/10.1002/cpt.1796) (2020)
* Zhang, J. D. *et al.* [Multiscale modelling of drug mechanism and safety](https://doi.org/10.1016/j.drudis.2019.12.009) (2019).
* Raza, U. *et al.* [MicroRNAs: master regulators of drug resistance, stemness, and metastasis](https://pubmed.ncbi.nlm.nih.gov/24509937/) (2014).
* Aigner, S. *et al.* [Human pluripotent stem cell models of autism spectrum disorder: emerging frontiers, opportunities, and challenges towards neuronal networks in a dish](https://pubmed.ncbi.nlm.nih.gov/24232378/) (2014).

<p id="preprints"></p>
<a href="#top" style="font-size: 11px;">Back to top</a>
#### Preprints

* Rüegger, J. *et al.* [CellEKT: A robust chemical proteomics workflow to profile cellular target engagement of kinase inhibitors](https://www.biorxiv.org/content/10.1101/2024.10.01.616061v1) (2024)
* Valeanu, A. *et al.* [Kinex infers causal kinases from phosphoproteomics data](https://www.biorxiv.org/content/10.1101/2023.11.23.568445v1) (2023)
* Geser, P. and Zhang, J. D. [Gene symbol recognition with GeneOCR](https://www.biorxiv.org/content/10.1101/2022.07.01.498459v3) (2022)
* Zhang, Y. *et al.* [A Novel, Anatomy-Similar in Vitro Model of 3D Airway Epithelial for Anti-Coronavirus Drug Discovery](https://doi.org/10.1101/2021.03.03.433824) (2021)
* Fang, T. *et al.* [Gene-set enrichment with regularized regression](https://www.biorxiv.org/content/10.1101/659920v2) (2019).

<p id="patents"></p>
<a href="#top" style="font-size: 11px;">Back to top</a>
#### Patents

* [Oligonucleotides for modulating RTEL1 expression](https://patents.google.com/patent/MX2021000404A/en) (2020)
* [Pyrrolo\[2,3-b\]pyrazine compounds as cccDNA inhibitors for the treatment of Hepatitis B Virus (HBV) infection](https://patents.google.com/patent/EP3720441A1) (2018)

<p id="book"></p>
<a href="#top" style="font-size: 11px;">Back to top</a>
#### Book chapters

* Edited by Adetayo Kasim, S. H., Ziv Shkedy, Sebastian Kaiser & Talloen, W.
   Applied Biclustering Methods for Big and High-Dimensional Data Using R
   (Chapman & Hall/CRC Biostatistics Series). Chapman and Hall/CRC (2016).
   ([Publisher's
   Link](https://www.routledge.com/Applied-Biclustering-Methods-for-Big-and-High-Dimensional-Data-Using-R/Kasim-Shkedy-Kaiser-Hochreiter-Talloen/p/book/9781482208238),
   [Amazon](https://www.amazon.com/Applied-Biclustering-Methods-High-Dimensional-Biostatistics/dp/1482208237)).


<p id="conference"></p>
<a href="#top" style="font-size: 11px;">Back to top</a>
#### Conference posters and papers

* Bosch, M *et al.* [OS-032 A PKA-associated liver-tissue rheostat curbs T cell receptor signalling and effector function of virus-specific CD8 T cells in chronic viral hepatitis](https://www.easlcongress.eu/wp-content/uploads/2023/06/EASL_2023_Congress_Abstracts_version5_20June_compressed.pdf). **EASL** (European Association for the Study of the Liver)/ **Journal of Hepatology** (2023)
* Klughammer, B *et al.* [P805 Selnoflast, a potent NLRP3 inhibitor - results from a phase 1b experimental medicine study in patients with Ulcerative Colitis](https://academic.oup.com/ecco-jcc/article/17/Supplement_1/i938/7010202). **Journal of Crohn's and Colitis** (2023)
* Zhou, X *et al.* [PS-156-Antigen suppression effect of the core protein allosteric modulator R07049389 in AAV-HBV mice is accompanied by transient upregulation of immune-cell gene signatures in the liver](https://doi.org/10.1016/S0618-8278\(19\)30174-4). Journal of Hepatology (2019)
* Ward, A. *et al.* Abstract A14: Re-expression of microRNA-375 reverses both tamoxifen resistance and accompanying EMT-like properties in breast cancer (2012).

<p id="essays"></p>
<a href="#top" style="font-size: 11px;">Back to top</a>
#### Essays

* [How young people benefit from Swiss apprenticeships](https://www.nature.com/articles/d41586-024-01102-8), **Nature**, 2024.

<p id="software"></p>
<a href="#top" style="font-size: 11px;">Back to top</a>
#### Software

Here is a selection of the scientific software that I developed, often with help
and support of many other colleagues. More open-source software can be found in
my Github account [Accio](http://github.com/Accio) as well as the public Github
account of my team [BEDApub](http://github.com/BEDApub).

* [BESCA](https://github.com/bedapub/besca): The [BESCA](https://github.com/bedapub/besca) package in Python was developed by team members of the Bioinformatics and Exploratory Data Analysis team to support single-cell omics data analysis. It is particularly useful in translational studies: the analysis with BESCA has informed key decisions in multiple pre-clinical and clinical projects.
* [BioQC](https://www.bioconductor.org/packages/release/bioc/html/BioQC.html): [BioQC](https://www.bioconductor.org/packages/release/bioc/html/BioQC.html) performs quality control of high-throughput expression data based on tissue gene signatures. It can detect tissue heterogeneity in gene expression data. The core algorithm is a Wilcoxon-Mann-Whitney test that is optimised for high performance.
* [ddCt](The ddCt Algorithm for the Analysis of qRT-PCR, or quantitative real-time PCR): The Delta-Delta-Ct (ddCt) Algorithm is an approximation method to determine relative gene expression with quantitative real-time PCR (qRT-PCR) experiments. Compared to other approaches, it requires no standard curve for each primer-target pair, therefore reducing the working load and yet returning accurate enough results as long as the assumptions of the amplification efficiency hold. The ddCt package implements a pipeline to collect, analyse and visualize qRT-PCR results, for example those from TaqMan SDM software, mainly using the ddCt method. The pipeline can be either invoked by a script in command-line or through the API consisting of S4-Classes, methods and functions.
* [HBVouroboros](https://github.com/bedapub/HBVouroboros) is written in Python and implements a Snakemake-based workflow to map Illumina RNA-sequencing reads to HBV cccDNA. It offers following functionalities: (a) mapping reads to reference genomes of of eight major HBV genotypes (A-H). (b) *de novo* assembly of the HBV genome, (c) inference of the reference strain from which the reads are likely generated and the genotype, and (d) reporting base-level, gene-level, and HBV-genome-level quantification of read counts, as well as structural variants with regard to the inferred reference strain.
* [Kinex](https://github.com/bedapub/kinex) infers causal serine/threonine kinases from phosphoproteomics data. Kinex users can score kinase-substrate interactions, perform enrichment analysis, visualise candidates of causal regulators, and query similar profiles in a database of drug-induced kinase activities. The software was mainly developed by Alexandra Valeanu, an intern sponsored by Roche Advanced Analytics Network (RAAN).
* [KEGGgraph](https://www.bioconductor.org/packages/release/bioc/html/KEGGgraph.html) is an interface between KEGG pathway and graph object as well as a collection of tools to analyze, dissect and visualize these graphs. It parses the regularly updated KGML (KEGG XML) files into graph models maintaining all essential pathway attributes. The package offers functionalities including parsing, graph operation, visualization and etc.
* [ribiosUtils](https://cran.r-project.org/web/packages/ribiosUtils/index.html) is a R package that provides interface to the Bioinfo-C (internal name: BIOS) library and utilities. 'ribiosUtils' is a swiss-knife for computational biology in drug discovery, providing functions and utilities with minimal external dependency and maximal efficiency. I develop and maintain this software package in memory of Clemens Broger.
* [RpsiXML](https://www.bioconductor.org/packages/release/bioc/html/RpsiXML.html) queries, data structure and interface to visualization of interaction datasets. This package inplements the PSI-MI 2.5 standard and supports up to now 8 databases. Further databases supporting PSI-MI 2.5 standard will be added continuously.
* [rqubic](https://www.bioconductor.org/packages/release/bioc/html/rqubic.html) implements the QUBIC algorithm introduced by Li et al. for the qualitative biclustering with gene expression data.
* [RTCA](https://www.bioconductor.org/packages/release/bioc/html/RTCA.html) is able to import, analyze and visualize data from Roche(R) xCELLigence RTCA systems. The package imports real-time cell electrical impedance data into R. As an alternative to commercial software shipped along the system, the Bioconductor package RTCA provides several unique transformation (normalization) strategies and various visualization tools.

