---
layout: post
title: MicroRNAs - Small RNA, Big Prize
category: biology
tags: microRNA Nobel
---

My sincere congratulations go to Victor Ambros and Gary Ruvkun, who won the Nobel Prize in Physiology or Medicine 2024 for the discovery of microRNA and its role in post-transcriptional gene regulation ([source](https://www.nobelprize.org/prizes/medicine/2024/summary/)). The news reminds me of my PhD time, when I became fascinated by microRNAs.

A major part of my PhD work was about inferring functions of microRNA, studying how microRNA regulates gene expression in diseases such as breast cancer, and using microRNA as diagnostic tools. In particular, together with my colleagues, we discovered that two very similar clusters of microRNAs, miR-200bc/429 and miR-200a/141, have [distinct effects on proliferation and invasion of breast cancer cells](https://www.nature.com/articles/onc2010201). Later studies further revealed that [these miRNAs are up-regulated in human breast tumors compared with normal tissues](https://www.nature.com/articles/s41598-021-82286-1). Interestingly, in most aggressive molecular subtypes including Luminal B, HER2 and triple negative, which is associated with increased epithelial-to-mesenchymal transition (EMT) and invasion, their expression is again reduced. A landmark discovery by [Ambros and Ruvkun](https://www.nobelprize.org/prizes/medicine/2024/advanced-information/) was that microRNAs, often being promiscuous regulators of many proteins, are themselves tightly regulated during developmental phases, for instance in the nematode worm *Caenorhabditis elegans* (C. elegans). It seems that such phase-specific expression patterns also apply to tumor development.

Motivated by our initial work with microRNAs, my colleague and mentor Özgür Sahin designed and performed a miRome-wide proteomics screening, where we over-expressed one human microRNA a time and measured how two dozens of proteins change their expression in breast cancer cells. [Our work](https://www.embopress.org/doi/full/10.1038/msb.2011.100) become one of the first studies that describe the network of miRNA-protein regulation at the global level. By developing and applying a new network-analysis methodology, we detected consistent and intrinsic regulatory patterns where miRNAs simultaneously co-regulate several proteins that act in the same functional module. While it was already known to the field that miRNAs with similar sequences often target a same set of proteins, our observation further suggested that even dissimilar miRNAs cooperate to target proteins of similar functions, thereby reinforcing the effect of each other. It's like pointing several guns against a gang of several criminals: the co-regulation makes it more likely that the cell is 'locked-in' in a state.

Do miRNAs, like siRNAs (small interference RNAs), have the potential to become drugs? The [*News and View* article](https://www.embopress.org/doi/full/10.1038/msb.2012.2) associated with our study, authored by Marcos Malumbres and titled *miRNAs versus oncogenes: the power of social networking*, published in 2012, made the interesting observation the fact that AKT1 and ERK2, two major kinases in the PI3K and RAS oncogenic pathways, may be co-downregulated by 30 miRNAs suggests that targeting miRNAs (including targeting pairs of them) may become new therapeutics. In 2017, [Rupaimoole and Slack](https://www.nature.com/articles/nrd.2016.246) reviewed development of miRNA-based therapeutics, particularly in oncology, and reported that several therapeutics have already advanced into clinical testing. However, [a review by Kim and Croce in 2023](https://www.nature.com/articles/s12276-023-01050-9), notes that until then no miRNA-based therapies have been approved. This is a stark contrast to siRNA (for which Andrew Z. Fire and Craig C. Mello were awarded the Nobel Prize in Physiology or Medicine 2006), which has made into several drugs including Patisiran and givosiran. [Zhang et al., 2021](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7910153/) argues that this is probably due to the fact that one miRNA generally targets tens or hundreds of genes. The too-many-targets-for-miRNA-effect, according to the author, poses a higher hurdle with regard to safety.

How does it look in the pipelines now? A quick check on [ClinicalTrials](https://clinicaltrials.gov/search?intr=miRNA&aggFilters=phase:3%202%201,status:rec) revealed two ongoing, recruiting Phase I-II trials using microRNAs as therapeutics: [AMT-162](https://clinicaltrials.gov/study/NCT06100276), an artificial microRNA targeting the SOD1 gene for SOD1 amyotrophic lateral sclerosis (Phase 1/2); and [ATX-01](https://clinicaltrials.gov/study/NCT06300307), an anti-miR that inhibitors the microRNA miR-23b (Phase 1/2a). I am excited to learn the outcome of these and other finished trials.

I feel a bit shamed to confess that I did not know the names of Victor Ambros and Gary Ruvkun before, despite the fact that I worked with miRNAs for many years. From [the scientific background provided by the Nobel prize committee](https://www.nobelprize.org/prizes/medicine/2024/advanced-information/), I could recognize the name of David Bartel (whose lab developed the much appreciated database of miRNAs and their targets, known as [TargetScan](https://www.targetscan.org/vert_80/). Though I was familiar with the stories that miRNA was first identified from C. elegans, by reading [the press release](https://www.nobelprize.org/prizes/medicine/2024/press-release/) and the [scientific background](https://www.nobelprize.org/prizes/medicine/2024/advanced-information/), I learned about the amazing story of how Ambros and Ruvkun connected with each other and recognized the complematarity of sequences of the first-discovered miRNA, *lin-4*, and the protein-encoding mRNA, *lin-14*.

The discoveries made by Ambros, Ruvkun, and many other pioneers of microRNA biology revealed that by base-pair matching mechanism, microRNA binds to untranslated regions at the 3'-end of mRNA (3'-UTR). Starting from there, many researchers, most but few names of which remain unfortunately unknown to me, have built a huge branch of knowledge about how microRNAs expand and function during evolution, and how they are regulated in development and disease. The short, sometimes partial, yet always elegant matching of 21-23 nucleotides between miRNAs and mRNAs keeps fascinating me, after so many years.

